src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/Aapt2Tests.cs

```diff
@@ -84,6 +85,7 @@ public void Aapt2Link ()
 		}
 
 		[Test]
+		[Category ("SmokeTests")]
```

Comment:
The two tests in this file were fast, too: 410ms and 86ms.

Reply:
My thinking was that we may want _some_ task only test in the smoke test category to make sure that aspect of the framework remains functional. I think these can be removed in favor of the ManagedResourceParserTest above though.

Reply:
Maybe I don't understand. Which builds will run the `SmokeTest` category and which run everything?

Reply:
The Windows Build + Build Tree tests job will only run `SmokeTests`, while our macOS and Windows test jobs which run against our .pkg and .vsix files will run the full suite.